app/controllers/bills_controller.rb

```diff
@@ -45,4 +79,68 @@ def find_bill
     flash[:alert] = t "errors.bill.not_found"
     redirect_to root_path and return
   end
+
+  def create_stripe_session line_items
+    @line_items = load_lines_item(line_items)
+    @metadata = build_metadata(line_items)
+
+    @session = Stripe::Checkout::Session.create({
+      payment_method_types: %w(card), # rubocop:disable Layout/FirstHashElementIndentation
+      line_items: @line_items,
+      metadata: @metadata,
+      payment_method_options: {card: {request_three_d_secure: "any"}},
+      mode: "payment",
+      success_url: bills_url,
+      cancel_url: bills_url
+                                                })
```

Comment:
```suggestion
    @session = Stripe::Checkout::Session.create({
                           payment_method_types: %w(card), # rubocop:disable Layout/FirstHashElementIndentation
                           line_items: @line_items,
                           metadata: @metadata,
                           payment_method_options: {card: {request_three_d_secure: "any"}},
                           mode: "payment",
                           success_url: bills_url,
                           cancel_url: bills_url
                         })
```
Như này mới đúng chứ

Reply:
dạ rubocop nó bắt là ({ với }) thẳng hàng ạ